openpype/hosts/maya/plugins/load/load_reference.py

```diff
@@ -158,3 +165,17 @@ def _post_process_rig(self, name, namespace, context, options):
                 options={"useSelection": True},
                 data={"dependencies": dependency}
             )
+
+    @staticmethod
+    def _organize_containers(nodes):
+        # type: (list) -> None
+        for node in nodes:
+            id_attr = "{}.id".format(node)
+            if not cmds.attributeQuery("id", node=node, exists=True):
+                print("-" * 80)
+                print("skipping {}".format(node))
+                continue
+            if cmds.getAttr(id_attr) == AVALON_CONTAINER_ID:
+                print("=" * 80)
```

Comment:
Should this printing be logging instead? Seems quite verbose?

Reply:
leftover debug prints that shouldn't be there, removed.